requirements.txt

```diff
@@ -0,0 +1,27 @@
+appnope==0.1.0
+argh==0.26.2
+colorama==0.3.7
+coverage==4.2
+decorator==4.0.10
+docopt==0.6.2
+ipython==5.1.0
+ipython-genutils==0.1.0
+pathtools==0.1.2
+pexpect==4.2.0
+pickleshare==0.7.4
+pluggy==0.3.1
+prompt-toolkit==1.0.6
+ptyprocess==0.5.1
+py==1.4.31
+Pygments==2.1.3
+pytest==2.9.2
+pytest-cov==2.3.1
+pytest-watch==4.1.0
+PyYAML==3.11
+simplegeneric==0.8.1
+six==1.10.0
+tox==2.3.1
+traitlets==4.2.2
+virtualenv==15.0.3
+watchdog==0.8.3
+wcwidth==0.1.7
```

Comment:
The vast majority of these "requirements" are actually needed only for your tests.  If your code doesn't really require them to be installed, I would not add a requirements.pip file.